src/buildQuestions.js

```diff
@@ -51,13 +51,23 @@ const buildAutoCompleteChoices = (name, choices, mandatory) => {
     );
   }
   let minValueLength = 1;
+  let extensionFlg = false;
   _.forEach(choices, (value, title) => {
+    extensionFlg = _.isArray(value);
```

Comment:
I don't think this code is needed. 
But here are some comments about the code itself:
1. a side-effect and an anti-pattern: don't use a single value inside an array/map to indicate something about all the objects in that array/map. For example, in this case the last iteration will set the extensionFlg to false/true for the entire object.
2. if you want to support 2 different ways for this API, it should be done in the iteration itself (inside the _.map in this case), so that developers can use both syntaxes.
3. The variable extensionFlag is not explaining what it means

Thanks again for your contribution